### PR TITLE
Add missing properties to TooltipOptions interface

### PR DESCRIPTION
**Pull request by CloudPower97**
# Description

This `P.R` addresses #1113 

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

# How Has This Been Tested?

--

# Checklist:

- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [x] I have not generated a new package version. (the maintainers will handle that)

### CODE REVIEW

File: types/components/Button.d.ts [hunk 1]
```diff
@@ -10,10 +10,16 @@ export interface FabOptions {
 }
 
 export interface TooltipOptions {
+  exitDelay: Number;
+  enterDelay: Number;
+  html: string;
+  margin: Number;
+  inDuration: Number;
+  outDuration: Number;
+  transitionMovement: Number;
```
Comment (alextrastero) [inline types/components/Button.d.ts:19]: Should these properties be required?

Comment (alextrastero) [reply types/components/Button.d.ts:19]: `number` in lowercase

Comment (alextrastero) [reply types/components/Button.d.ts:19]: If you open `types/sample.test.tsx` you can test these types

Comment (CloudPower97) [reply types/components/Button.d.ts]: @alextrastero Can you review this once more? Everything should be fine now

**Comment by CloudPower97** [comment]
@alextrastero Here you go :wink:

**Pull request merged**